pages/account/statistics.js

```diff
@@ -97,7 +97,7 @@ export async function getServerSideProps(context) {
   }
 
   const last30DaysStats = last30Days.map(date => ({
-    views: dailyStatsMap.has(date) ? dailyStatsMap.get(date).views : 0,
+    views: dailyStatsMap.has(date) ? dailyStatsMap.get(date).views || 0 : 0,
```

Comment:
with this extra change, can this be simplified further with just ?

```suggestion
    views: dailyStatsMap.get(date).views || 0
```

Reply:
yes